Lib/sndhdr.py

```diff
@@ -49,20 +49,28 @@
 SndHeaders.sampwidth.__doc__ = ("""Either the sample size in bits or
 'A' for A-LAW or 'U' for u-LAW.""")
 
+
 def what(filename):
     """Guess the type of a sound file."""
-    res = whathdr(filename)
-    return res
+    try:
+        res = whathdr(filename)
+        return res
+    except (FileNotFoundError) as e:
```

Comment:
Please remove the extra ``()`` and unused ``as e``.

Reply:
Why silence FileNotFoundError at first point?